src/android/app/src/main/java/org/citra/citra_emu/features/settings/ui/SettingsFragmentPresenter.kt

```diff
@@ -759,7 +759,7 @@ class SettingsFragmentPresenter(private val fragmentView: SettingsFragmentView)
                     R.string.factor3d,
                     R.string.factor3d_description,
                     0,
-                    100,
+                    400,
```

Comment:
Wouldn't be better to modify the depth scale based on the immersiveModeFactor?

I imagine some users would complain if they lose stereo vision when switching immersive mode on/off.

Reply:
I think that's valid. I'm in favor of adding an offset, personally, but it's not a requirement for the change